documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -8,51 +8,103 @@ weight: 4
 
 ### Supported environments
 
-WebLogic Server and the operator are supported on
-Oracle cloud environments such as
-Oracle Cloud Infrastructure (OCI),
-Oracle Linux Cloud Native Environment (OLCNE),
-Oracle Private Cloud Appliance (PCA),
-or Oracle Private Cloud at Customer (OPCC),
-and on "Authorized Cloud Environments" as defined in
-[this Oracle licensing policy](https://www.oracle.com/assets/cloud-licensing-070579.pdf)
-for [this list of eligible products](http://www.oracle.com/us/corporate/pricing/authorized-cloud-environments-3493562.pdf).
+WebLogic Server and the operator are supported on Oracle Cloud offerings such as:
+- Oracle Cloud Infrastructure (OCI)
+- Oracle Container Engine for Kubernetes (OKE)
+- Oracle Linux Cloud Native Environment (OLCNE)
+- Oracle Private Cloud Appliance (PCA)
+- Oracle Private Cloud at Customer (OPCC)
 
-The official document that defines the supported configurations
-is [here](https://www.oracle.com/middleware/technologies/ias/oracleas-supported-virtualization.html)
-(search for keyword 'Kubernetes').
+WebLogic Server and the operator are certified on offerings such as:
+- Amazon Elastic Compute Cloud (EC2)
+- Microsoft Azure Platform
+- Microsoft Azure Kubernetes Service (AKS)
+- OpenShift Container Platform
+- VMWare Tanzu
+- VMware Tanzu Kubernetes Grid (TKG)
+
+WebLogic Server and the operator are also supported on service offerings which 
+deploy the WebLogic Server and the operator for you. These include:
+- Oracle WebLogic Server for OKE (WLS for OKE)
+- Oracle WebLogic Server on the Azure Kubernetes Service Marketplace (WLS on AKS Marketplace)
+
+[Development-focused Kubernetes distributions](#development-focused-kubernetes-distributions) are also supported.
+
+For detailed virtualization and cloud licensing discussions, see:
+- [Supported Virtualization Technologies for Oracle Fusion Middleware](https://www.oracle.com/middleware/technologies/ias/oracleas-supported-virtualization.html) (search for keyword 'Kubernetes')
+- [Running and Licensing Oracle Programs in Containers and Kubernetes](https://www.oracle.com/a/tech/docs/running-and-licensing-programs-in-containers-and-kubernetes.pdf)
 
 Some supported environments have additional help or samples that are specific
 to the operator, or are subject to limitations and restrictions: see
 [Important notes about specific environments](#important-notes-about-specific-environments).
 
-### Kubernetes, WebLogic, and operating system prerequisites
+### Kubernetes, WebLogic Server, and operating system prerequisites
 
-The operator is subject to Kubernetes, WebLogic, and operating system versioning prerequisites:
+The operator is subject to Kubernetes, WebLogic Server, and operating system versioning prerequisites:
 see [Operator prerequisites]({{< relref "/userguide/prerequisites/introduction.md" >}}).
 
 ### Pricing and licensing
 
+The WebLogic Kubernetes Operator and Oracle Linux are open source and free;
```

Comment:
Instead of free, I prefer to say "gratis".  See Stallman.

Reply:
I'll leave this up to @rosemarymarano, but I prefer "free" given that English is often not our target reader's first language.

Reply:
Yes, let's stick with "free". (We are directed to avoid Latin abbreviations, which doesn't exactly apply here, but it is the better choice for translation.)

Reply:
You're outvoted Ed. Please "resolve" if you yield.

Reply:
No problem. Free as in beer.

Reply:
Hah! But it is after 4PM here. Hmm...